config/settings/base.py

```diff
@@ -57,6 +57,10 @@
 # https://docs.djangoproject.com/en/stable/ref/settings/#std:setting-DEFAULT_AUTO_FIELD
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+# Demand data
+DEMAND_DIR = os.path.join(BASE_DIR, "static", "parquet")
```

Comment:
```suggestion
DEMAND_DIR = os.path.join(APPS_DIR, "static", "parquet")
```

When I tried it didn't find the file

Reply:
I didn't actually get to testing this code yet so I probably just took the wrong path. Thanks for the amendment :)